modules/sno-du-enabling-workload-partitioning.adoc

```diff
@@ -48,3 +48,33 @@ spec:
         user:
           name: root
 ----
+
+* The contents of `/etc/crio/crio.conf.d/01-workload-partitioning` should look like this.
```

Comment:
Replace the period with a colon, per the ISG here: https://docs.engineering.redhat.com/display/CCSENABLE/Colons+and+semicolons